### ISSUE TITLE
Implement Miller-Rabin Method in Primality Test Algorithms

### ISSUE THREAD

**Issue by adithyaakrishna** [deutranium/Algorithms#79]
Implement Miller-Rabin Method in Primality Test Algorithms in a language of your choice.

Please read the contributing guidelines in [CONTRIBUTING.md](https://github.com/deutranium/Algorithms/blob/master/CONTRIBUTING.md)

Languages claimed/implemented:
- [ ] C++
- [ ] Java
- [ ] Python
- [ ] C#

**Comment by karthikshetty03** [comment]
@adithyaakrishna , Can I work on this issue ?

**Comment by adithyaakrishna** [comment]
@karthikshetty03 I don't have the permission to assign you, but you can work on this issue and create a commit in your fork and once @deutranium 
assigns you, you can immediately create a PR :)

**Comment by deutranium** [comment]
@adithyaakrishna thanks for your helpful comments!!

@uravgkarthik Github requires you to comment on the issue first for me to be able to add you as an assignee. Please comment here with the language you would like to work with.

**Comment by 96Octavian** [comment]
If there's no assignee, I can provide a Python implementation and maybe translate it to C# (and probably Java too)

**Comment by deutranium** [comment]
@96Octavian sure. Should I assign all the three (C, Python, C#) to you?

**Comment by 96Octavian** [comment]
Yes please. I can do Python, C# and Java (I don't know a thing about C++ though, can't promise)

**Comment by deutranium** [comment]
@96Octavian sure I'll assign these 3 to you.
Happy coding!!

**Comment by arnav-roy** [comment]
I want to contribute in C++

**Comment by deutranium** [comment]
@arnav-roy sure assigned it to you

**Comment by 96Octavian** [comment]
Should camel case be followed even in languages where it is not standard naming convention?

**Comment by deutranium** [comment]
@96Octavian no you can follow for example snake case in python. That rule was added to ensure that there is a uniformity in code style
